cmd/serve.go

```diff
@@ -105,6 +110,11 @@ func (c *serveCmd) Command() *cobra.Command {
 				return fmt.Errorf("resolving distribution account signature client: %w", err)
 			}
 			cfg.DistributionAccountSignatureClient = signatureClient
+			appTracker, err := sentry.NewSentryTracker(sentryDSN, stellarEnvironment, 5)
+			if err != nil {
+				return fmt.Errorf("initializing App Tracker: %w", err)
+			}
+			cfg.AppTracker = *appTracker
```

Comment:
Here you should pass the reference because it's an interface!

Reply:
I made it
cfg.AppTracker = appTracker